src/config.rs

```diff
@@ -339,7 +339,7 @@ impl Config {
                 .get_from(Some("defaults"), "org")
                 .map(str::to_owned)
                 .ok_or_else(|| {
-                    format_err!("An organization slug is required (provide with --org)")
+                    format_err!("An organization id or slug is required (provide with --org)")
```

Comment:
```suggestion
                    format_err!("An organization ID or slug is required (provide with --org)")
```

Please update anywhere else you have used lowercase `id`.